addons/website/controllers/main.py

```diff
@@ -401,6 +401,7 @@ def get_dynamic_snippet_templates(self, filter_name=False):
             t['rowPerSlide'] = attribs.get('data-row-per-slide')
             t['arrowPosition'] = attribs.get('data-arrow-position')
             t['extraClasses'] = attribs.get('data-extra-classes')
+            t['thumb'] = attribs.get('data-thumb')
```

Comment:
Review all commits one by one: when you edit a commit message, you have the list of apps you touched. I can't see a commit message saying "website_sale" and have the first change in "website" :wink: